lib/fluent/plugin/in_windows_eventlog.rb

```diff
@@ -55,6 +59,20 @@ def configure(conf)
                           else
                             method(:no_encode_record)
                           end
+      if @store_pos
+        unless system_config.root_dir
+          raise Fluent::ConfigError, "windows_eventlog: 'root_dir' in <system> parameter must be specified with 'store_pos' parameter."
+        end
+        @chs.map {|ch|
+          config = Fluent::Config::Element.new('storage',
+                                               "in_windows_eventlog_pos_#{ch.gsub(' ', '_')}", {
+                                                 "@type" => "local",
+                                                 "persistent" => true,
+                                               }, [])
+          @storages[ch] = storage_create(usage: "in_windows_eventlog_pos_#{ch.gsub(' ', '_')}", conf: config,
```

Comment:
It's not good enough to use this style method call for `storage_create` (it prevents users to write their own configuration section as `<storage channel>`).
I think it works well to use just `usage` with `config_section` to specify default parameter (for `@type` and `persistent`).

```ruby
config_section :storage do
  config_set_default :@type, DEFAULT_STORAGE_TYPE
  config_set_default :persistent, true
end

def configure(conf)
  # ...
  @chs.each do |ch|
    escaped_ch = ch.gsub(/[^0-9a-zA-Z]/, '_')
    @storages[ch] = storage_create(usage: escaped_usage)
  end
end
```